packages/aws-rfdk/lib/core/scripts/bash/mountFsxLustre.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This script will mount an Amazon FSx for Lustre File System to a specified mount directory on this instance,
+# and set up /etc/fstab so that the file system is re-mounted on a system reboot.
+#
+# This script requires the Lustre client to be already installed on the instance.
+# See https://docs.aws.amazon.com/fsx/latest/LustreGuide/install-lustre-client.html
+#
+# Note: This script uses get_metadata_token and get_region from ./metadataUtilities.sh
+#  Thus, the system must have applications pre-installed as outlined in that file.
+#
+# Script arguments:
+#  $1 -- file system Identifier (ex: fs-00000000000)
+#  $2 -- Mount path; directory that we mount the file system to.
+#  $3 -- Mount name
+#  $4 -- (optional) NFSv4 mount options for the file system.
```

Comment:
These should be lustre mount target options rather than NFSv4, I think

Reply:
Yeah another copy-paste, thanks for catching it! Updated